moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -41,10 +41,11 @@
 #include <boost/filesystem/path.hpp>        // for creating folders/files
 #include <boost/filesystem/operations.hpp>  // is_regular_file, is_directory, etc.
 #include <boost/algorithm/string/trim.hpp>
+#include <tinyxml.h>
```

Comment:
I don't see where you used this?  Is it needed and somehow in ros1 it came with some other header?